src/ser.rs

```diff
@@ -246,26 +246,31 @@ mod test {
     use serde_json::{from_str, to_string};
 
     proptest! {
+        #[cfg_attr(miri, ignore)]
```

Comment:
Could you explain in a comment somewhere the criterion for excluding a test from miri? Also, if you want to exclude a whole `mod` then maybe just `#[cfg(not(miri))]` at the top is easier instead of annotating every test.

Reply:
The logic behind not using `cfg(not(miri))` is that it causes a bunch of 'unused import' warnings, and with ignore it's possible to use `--ignored` to run it anyways, if you really want to try waiting for it to complete.

I'll add some documentation when I get a minute. For posterity, it's basically just 'takes more than ~30 seconds to run', and some tests may actually prefer running but with reduced iteration values. I just didn't want to take the time to track down why the specific iteration numbers were chosen for many of the issue tests, to find out whether they could be reduced and still have the test be meaningful.